physics/sfc_diag.f

```diff
@@ -1,39 +1,62 @@
 !>  \file sfc_diag.f
```

Comment:
I'm not sure what is going on with the changes in this file, but it looks like it's missing changes from previous commits. See https://github.com/ufs-community/ccpp-physics/commit/6f1c2b1188737721b30bf4a98e432792a1c24742#diff-5210406169e53c367678af29f850545c25156baf7a3f70ab811ea9adba8b7cae. I wonder if you could start from the latest version in ufs/dev and put your changes back in?

Reply:
There was some confusion with this code. It is ok now.